app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/email/CheckYourEmailController.scala

```diff
@@ -164,6 +169,14 @@ class CheckYourEmailController @Inject() (
               "[CheckYourEmailController][sendVerification] - " +
                 "Unable to send email verification request. Service responded with 'already verified'"
             )
+            if (service.enrolmentKey == Service.cds.enrolmentKey)
```

Comment:
**If** without **else** is not a recommended pattern to use in Scala as a return type of such call is Any (undefined): https://stackoverflow.com/questions/45619279/what-is-the-return-type-of-a-scala-if-statement-without-else.